src/btree/bt_vrfy.c

```diff
@@ -355,6 +367,11 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
             WT_ERR(__dump_layout(session, vs));
     }
 
+    /* Report WT_NOTFOUND if a checkpoint to verify was specified but not found. */
+    if (cfg_ckpt.len > 0 && !check_done) {
+        ret = WT_NOTFOUND;
+    }
```

Comment:
We don't use curly brackets for one-liners.
```suggestion
    if (cfg_ckpt.len > 0 && !check_done)
        ret = WT_NOTFOUND;
```

Reply:
Done, should we add an `s_` check for this in the `dist` folder? Something like `s_brackets`? Or maybe look for a clang-format setting that handles it instead. WDYT?

Reply:
Yes that would be great!